lib/components/viewers/amenities-panel.tsx

```diff
@@ -133,7 +135,7 @@ class AmenitiesPanel extends Component<Props, State> {
           <li className="related-item" key={key}>
             <div className="item-label">
               <div className="overflow-ellipsis" title={label}>
-                {company.icon}
+                <span aria-hidden>{company.icon}</span>
```

Comment:
This is a good addition. I would also recommend adding it to the micro mobility icon (line 228)

Reply:
Good idea thanks!